src/core/layout/qgslayoutexporter.h

```diff
@@ -491,6 +493,8 @@ class CORE_EXPORT QgsLayoutExporter
     static ExportResult print( QgsAbstractLayoutIterator *iterator, QPrinter &printer,
```

Comment:
Ideally we'd leave this exposed, but raise a QgsNotSupportedException when called instead. See eg QgsFileUtils::driveType

Reply:
@nyalldawson , I initially wanted to do that (back when I migrated this class to QPdfWriter) but I'm not sure it's possible since there's a QPrinter function parameter.

Reply:
Oh, right you are! I missed that...